docs/source/transaction_family_specifications/supplychain_transaction_family.rst

```diff
@@ -0,0 +1,317 @@
+***************************************
```

Comment:
Specs are design docs.  We're trying to promote the proper way to develop transaction families: Write a spec, iterate on it, then write the transaction processor.  By doing this simultaneously, it's difficult to know which is actually correct: the spec or the code.